src/FieldWarning/Assets/UI/Ingame/SelectionManager.cs

```diff
@@ -245,18 +254,49 @@ private bool IsInside(Vector3 t)
 
         private void UnselectAll(List<PlatoonBehaviour> selectedPlatoons, bool justPreviewing)
         {
+            foreach (var wp in _selectedWaypointOverlays)
```

Comment:
I would not use 'var' here, it is not obvious what the type of this variable is.